src/components/profile/profile.component.tsx

```diff
@@ -52,45 +63,65 @@ export const Profile = ({
       return (
         <ProfileAvatarStyle
           src={src}
-          alt={getInitials()}
+          darkBackground={darkBackground}
+          alt={alt}
+          name={name}
           initials={getInitials()}
           size={size}
-          shape="circle"
           data-element="user-image"
         />
       );
     }
     return (
       <ProfileAvatarStyle
         initials={getInitials()}
+        darkBackground={darkBackground}
         gravatar={email}
+        alt={alt}
+        name={name}
         size={size}
       />
     );
   };
 
-  const text = () => {
-    return (
-      <ProfileDetailsStyle size={size} hasSrc={!!src}>
-        <ProfileNameStyle size={size} data-element="name">
-          {name}
-        </ProfileNameStyle>
-        <ProfileEmailStyle size={size} data-element="email">
-          {email}
-        </ProfileEmailStyle>
-      </ProfileDetailsStyle>
-    );
+  invariant(
```

Comment:
**comment** (non-blocking): I'd be cautious about using `invariant` here unless you have very clear requirements that consumers should absolutely not do this. If the condition fires the entire app will crash.

This isn't a breaking change (assuming consumers were using typescript), because both `name` and `email` were required props before - but even so I'd be personally more inclined to suggest this more gently, perhaps with a `console.warn`, because I don't personally see a good reason why you couldn't leave out `name` and just have the corresponding element  be empty or not there.

Reply:
Yeah fair point, having a name is pretty essential to the component as an email or text underneath really doesn't make much sense without it. I'll leave this one for the next reviewer to get their opinion if that's ok as i'm on the fence

Reply:
yeah that's fair enough, I just thought I'd ask the question. If we're sure it makes no sense to have text/email with no name then I'm happy to keep this.

Reply:
Yep agree invariant feels a little heavy handed here